src/index.js

```diff
@@ -39,6 +40,7 @@ module.exports = {
     wilayaProjection,
     isValidWilayaCode,
     isValidZipCode,
+    hasName,
```

Comment:
I think that it's okay to not export this function as we're only using it internally. If you see that it make sense to do it, we'd require to add a jsDoc to the function, and we'll need to update the README

Reply:
For now I see that it doesn't indeed , maybe for next usecases it can be usable but not for not as i know.

Reply:
So, let's not export it please